js/items/tree.js

```diff
@@ -0,0 +1,42 @@
+class Tree {
+  constructor(center, size, heightCoef = 0.3) {
+    this.center = center;
+    //size of the base
+    this.size = size;
+    this.heightCoef = heightCoef;
+  }
+
+  //priviate method to generate level to the tree
+  #generateLevel(point, size) {
+    const points = [];
+    const rad = size / 2;
+    //loop through each angle(pi / 16) in a circle up to (360 degrees)
+    for (let a = 0; a < Math.PI * 2; a += Math.PI / 16) {
+      //add variability to the radius of each point generated around the central point.
+      const kindOfRandom = Math.cos(((a + this.center.x) * size) % 17) ** 2;
```

Comment:
- Cosine: gives a value between [-1,1]
- **2 (square consine) gives a value between [0,1]
- a + this.center.x : each level of the tree is going to generated randomly because of the size - contribution as they are different
- this.center.x is also different because the center location is different
- % 17 : prime number and gives a range between [1,16]. This operation introduces - -- --- 
- periodicity and limits the range of possible values.